mopidy/mpd/translator.py

```diff
@@ -107,9 +107,6 @@ def track_to_mpd_format(track, position=None, stream_title=None):
 
     if track.album and track.album.uri:
         result.append(('X-AlbumUri', track.album.uri))
-    if track.album and track.album.images:
-        images = ';'.join(i for i in track.album.images if i != '')
-        result.append(('X-AlbumImage', images))
```

Comment:
Should we keep this and call `get_images()` from here?

Reply:
I have no idea what MPD does these days. Kinda depends on them to be honest.

Reply:
I've decided to not worry about this right now. We don't have easy access to the core actor here anyway, since we're inside the Track to MPD representation translator.